.build/tar/generate_tar.sh

```diff
@@ -1,5 +1,19 @@
 #!/bin/bash
 
+# Copyright 2020, Google Inc.
```

Comment:
Not sure if this is required. Collector repo doesn't add license clause to configs, e.g. https://github.com/open-telemetry/opentelemetry-collector/blob/master/examples/k8s.yaml

Also, some data formats don't support comments, e.g. JSON. So we may not be able to add it consistently everywhere.